packages/react/src/themes/teams/components/Chat/chatMessageVariables.ts

```diff
@@ -37,8 +39,10 @@ export default (siteVars): ChatMessageVariables => ({
   offset: pxToRem(100),
   padding: pxToRem(16),
   authorMarginRight: pxToRem(12),
-  authorFontWeight: siteVars.fontWeightBold,
+  authorColor: siteVars.gray02,
+  authorFontWeight: siteVars.fontWeightRegular,
   headerMarginBottom: pxToRem(2),
+  contentColor: siteVars.colors.grey[900],
```

Comment:
This hex value is actually slightly wrong, but it should be fixed with the new color palette updates (`#252423` instead of `#252424`. Can you add a comment here so we know that when the color palette updates go in we should be using `gray[750]`?

Reply:
Replaced with direct value and added a note 👍